Akutsangs-folder/recipes-assignment/index.html

```diff
@@ -4,23 +4,18 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/class-work/Akutsangs-folder/recipes-assignment/index.css">
-    <title> Odin-Recipes</title>
+    <link rel="stylesheet" href="index.css">
+    <title>Odin-recipes</title>
 </head>
 <body>
-    <div class="flex-container">
-        <div class="one">
-          <a href="dessert.html"> Dessert</a>
-        </div>  
-        <div class="two">
-          <a href="BBQ-grills.html">BBQ-grills</a>
-        </div>
-        <div class="three">
-          <a href="toast.html">Toast</a>
-        </div>
-        <div class="four">
-          <a href="chicken.html">Chicken</a>
-        </div>  
-    </div> 
+    <h1>welcome to our recipes page</h1>
+    <div class="container">
+        <ul>
+            <a href="/Akutsangs-folder/recipes-assignment/BBQ-grills">BBQ-grills</a>
```

Comment:
Good job! I think if you look at the absolute link to BBQ-grills recipe very well, you'll see that BBQ-grills isn't a HTML file probably that's why the link doesn't work.

Reply:
I think it will be nice if you look at your source address for images in all pages since all the images are broken and doesn't load.

Cheers and Happy coding!👏👏👏